dependencies.props

```diff
@@ -21,15 +21,15 @@
 
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
-    <CoreFxExpectedPrerelease>preview2-25319-04</CoreFxExpectedPrerelease>
-    <CoreClrExpectedPrerelease>preview2-25316-03</CoreClrExpectedPrerelease>
+    <CoreFxExpectedPrerelease>preview1-25320-01</CoreFxExpectedPrerelease>
+    <CoreClrExpectedPrerelease>preview1-25319-05</CoreClrExpectedPrerelease>
```

Comment:
That PR was merged 19 days ago - a fundamental issue with the change should have come up earlier. Have we looked into where is the ApiCompat tool looking for the API? In SPC.dll or some library built in CoreFX?